docs/src/css/components/_button.scss

```
@@ -0,0 +1,98 @@
+    pointer-events: none;
+
+    span {
+      opacity: 0;
+    }
+
+    &::after {
+      content: '';
+      position: absolute;
+      width: 1em;
+      height: 1em;
+      border: 2px solid currentColor;
```

Stylelint (17.10.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)